### DISCUSSION TITLE
Doesn't Artisan support Typhoon anymore?

### DISCUSSION THREAD

**Discussion by jonasskafte** [artisan-roaster-scope/artisan#844] [General]
Hi guys,

Just wondering if anyone knows whether Artisan has stopped supporting Typhoon roasters in the latest release (2.6.0)?

I suddenly can't find Typhoon anymore under Config → Machine. And I also can't find it here:

https://artisan-scope.org/machines/typhoon/

Hope someone can help solve this mystery!

Cheers!

**Comment by jonasskafte** [comment]
UPDATE:

I can see now in the changelog that support for Typhoon has indeed been removed:

```
removes support for Trobrat and Typhoon machines
removes Russian localization
```

I assume this has to do with supporting the sanctions against Russia, which I'm personally completely on board with and fully support!

However, could it perhaps be an idea to simply remove the Russian localization? In that way, you'll mainly target actual Russians using Artisan and not those roasteries who happen to have bought a Typhoon roaster and don't support Putin. In fact, when we bought our roaster we actually thought it came from Czechia as that's where their showroom and sales office is. Also, they recently fired a sales agent as there was doubt about their support for Putin – or at least because of poor communication. See here:

https://www.instagram.com/p/CamVcaVhH1n/

I think their post on Instagram makes it clear that Typhoon is against Putin and in support of Ukraine. And also, the fact that they are partly based in Czechia, makes them somewhat not fully Russian I'd say.

Anyway, without going too political, further pressure should of course be put on Putin, but it would be helpful, to those of us outside of Russia with a Typhoon machine, if we could continue using Artisan. I therefore hope you'll consider reimplementing support for Typhoon, but just take away the Russian localization.

Many thanks and all the best!

**Comment by jonasskafte** [comment]
On further reflection, whatever puts max pressure on Putin I'm in support of. So if this is how it has to be, then that's just how it has to be (I wish there was a different way though).

Keep up the good work! I hope we'll be able to use Artisan again one day!

**Comment by jonasskafte** [comment]
Hi there @MAKOMO, just a follow-up on this in case it'll change your stance on supporting Typhoon. Please see this Instagram post where they state that they've closed down their site in Russia:

[https://www.instagram.com/p/CeQbFNaKLbQ/](https://www.instagram.com/p/CeQbFNaKLbQ/)

Cheers!

**Comment by deviantworld** [comment]
Hey guys, any news on that? It is been months later and there should be no reason to stopp supporting typhoon roasters on artisan.

**Reply by MAKOMO** [reply]
Next release will support Typhoon machines again. It is scheduled to be released begin of October. In the meantime you can use the v2.6.1 pre-release with Typhoon support available here: https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous
